### ISSUE TITLE
Visuals do not match collision shape at highest LOD - 5

### ISSUE THREAD

**Issue by someThrowAway1900** [Zylann/godot_heightmap_plugin#421]
**Describe the bug**
Visuals do not match collision shape at highest LOD - 5.  My understanding is that the highest visual LOD should be shown closest to the player (camera), and the lowest LOD should be furthest away from the camera, but this is not the case?

**To Reproduce**
Please refer video

**Expected behavior**
At the highest LOD, I would expect the visuals to match the collision shape. In the editor, I think the highest LOD is being displayed, but this is not the case when I run the game.

**Screenshots**
I placed markers corresponding to where the mesh vertexes are in the editor to help illustrate the issue.
edit - I forgot to mention that I am refreshed the collision twice (~ 40s mark) in the video as per the documentation - "Reload while the game is running"
https://github.com/Zylann/godot_heightmap_plugin/assets/132544925/41f5472d-642f-43a8-a609-cd524d3dbfa8

**Environment**
- OS: Windows 10 Pro
- Graphics card: AMD 6600xt
- Godot version: 4.2.1 stable
- Plugin version: 1.7.3 dev
- Renderer used: Forward+

**Comment by Zylann** [comment]
Can you provide a minimal reproduction project showing this, instead of a video? It is the preferred way stuff like this should be reported because even though a video shows you have the issue, it doesn't provide me with the full exact setup you have that makes this issue happen.

The only guess I have at the moment, is that LOD is calculated based on camera position, but because your camera is above the terrain, it might be off from where you expect LOD to be centered on (the character). But I'm not sure because I would not imagine it to be that off.

**Comment by someThrowAway1900** [comment]
I couldn't replicate the issue in a new project, as everything was working correctly. I even tried exporting the height map from my main project, and it stilled worked.

In the end, I stumbled across the cause, and my fault, the issue arose because I moved the hTerrain node outside of the Node3D node (image left) per my scene tree. This is what caused the LOD issues. Moving the hTerrain node back under the Node3D (image right) resolved the issue.

Apologies for the user mistake. Closing issue.

![Capture](https://github.com/Zylann/godot_heightmap_plugin/assets/132544925/bde3b332-a23b-4b90-9049-d03dfa3c2a3b)
